Makefile

```diff
@@ -280,12 +280,7 @@ INSTALL_STUBLIBDIR=$(DESTDIR)$(STUBLIBDIR)
 INSTALL_MANDIR=$(DESTDIR)$(MANDIR)
 INSTALL_FLEXDLL=$(INSTALL_LIBDIR)/flexdll
 
-TOPINCLUDES=-I otherlibs/$(UNIXLIB) \
-  -I otherlibs/bigarray \
-  -I otherlibs/dynlink \
-  -I otherlibs/str \
-  -I otherlibs/$(GRAPHLIB) \
-  -I otherlibs/raw_spacetime_lib
+TOPINCLUDES=$(addprefix -I otherlibs/,$(filter-out %threads,$(OTHERLIBRARIES)))
```

Comment:
@gasche - the `filter-out` is responsible for removing `systhreads` and `threads` from the list.